### ISSUE TITLE
Model release

### ISSUE THREAD

**Issue by benedekrozemberczki** [Sinwang404/DeepDDs#1]
Is this the repository for "DeepDDS: deep graph neural network with attention mechanism
to predict synergistic drug combinations"? Are you going to release the code?

**Comment by Sinwang404** [comment]
Yes, this is the code of our paper(DeepDDS). The code was released to the master of the repository.
